src/widgets/buttons.ts

```diff
@@ -27,6 +27,16 @@ export type StatusAreaContext = {
   div?: HTMLElement
   dom?: HTMLDocument
 }
+export enum ButtonType {
+  Primary = '#7c4dff',
+  Secondary = '#01C9EA',
+  Cancel = '',
```

Comment:
You can see below there is a cancelButton and a continueButton that literally just call button with Cancel and Continue should we look at setting these styles?

Reply:
I thought the cancel and continue buttons use icons?
So yeah, we need to work out how we want to style icon-buttons.
In this PR it's fine if you look only at text buttons, not icon buttons.

Reply:
there's both actually.  icons and then buttons.  I don't think buttons are in the examples.

Reply:
See [button](https://solid.github.io/solid-ui/examples/buttons/#button), [cancelButton](https://solid.github.io/solid-ui/examples/buttons/#cancelButton), [continueButton](https://solid.github.io/solid-ui/examples/buttons/#continueButton)